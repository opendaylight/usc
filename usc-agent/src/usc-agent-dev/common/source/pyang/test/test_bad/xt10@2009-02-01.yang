module xt1x { // error: does not match filename
  namespace "urn:xt10";
  prefix "xt10";

  revision 2009-02-01 {
    description "second update";
  }

  revision 2009-01-01 {
    description "first update";
  }

  typedef a {
    type string;
  }

/*
  typedef b {
    type string;
  }
*/

  typedef c {
    type string;
  }

}
