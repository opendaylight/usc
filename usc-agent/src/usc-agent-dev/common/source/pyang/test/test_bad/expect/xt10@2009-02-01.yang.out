xt10@2009-02-01.yang:1: warning: WBAD_MODULE_NAME
